client/src/data/interviews/technical-screening.tsx

```diff
@@ -0,0 +1,488 @@
+  },
+  {
+    id: 'oop',
+    title: 'OOP (Encapsulation, Polymorphism, and Inheritance)',
+    topic: 'Computer Science',
+  },
+  {
+    id: 'html',
+    title:
+      "Position and display attributes' values, tags, weight of selectors, pseudo-classes and elements, box model, relative and absolute values, em vs rem, semantic, semantic tags, etc.",
+    topic: 'HTML/CSS',
+  },
+  {
+    id: 'array',
+    title: 'Array. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'list',
+    title: 'List. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'stack',
+    title: 'Stack. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'queue',
+    title: 'Queue. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'tree',
+    title: 'Tree. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'hashTable',
+    title: 'Hash table. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'heap',
+    title: 'Heap. Operations complexity.',
+    topic: 'Data structures',
+  },
+  {
+    id: 'structuresOverview',
+    title: 'Difference between list and array, or between stack and queue.',
+    topic: 'Data structures',
+  },
+];
+
+const theory: Step = {
+  id: FeedbackStepId.Theory,
+  title: 'Theory',
+  stepperDescription: 'Talk about theory, how things work',
+  description: (
+    <>
+      <div>
+        Ask student some questions from the self-study course. You can use the list of recommended questions or add your
+        own.
+      </div>
+      <div>
+        <ClockCircleOutlined />
+        &nbsp; 15-30 min
+      </div>
+    </>
+  ),
+  items: [
+    {
+      type: InputType.Rating,
+      id: 'questions',
+      title: 'What questions did the student have to answer?',
+      required: true,
+      tooltips: SKILLS_LEVELS,
+      questions: theoryQuestions,
+      examples: theoryQuestions,
+    },
+    {
+      id: 'comment',
+      type: InputType.TextArea,
+      title: 'Your comment',
+      placeholder: "Comment about student's skills",
+    },
+  ],
+};
+const practiceQuestions = [
+  {
+    id: '1',
+    title: `Given an integer array arr and a filtering function fn, return a new array with a fewer or equal number of elements. 
+  The returned array should only contain elements where fn(arr[i], i) evaluated to a truthy value.`,
+  },
+  {
+    id: '2',
+    title: `Implement a function that takes two arrays of numbers and returns an array of numbers that are common between the two input arrays.`,
+  },
+];
+const practice: Step = {
+  id: FeedbackStepId.Practice,
+  title: 'Practice',
+  stepperDescription: 'Propose technical tasks to solve',
+  description: (
+    <>
+      Ask the student to solve the coding problem. See the list of examples of coding tasks or suggest another problem
+      of the same level.
+      <div>
+        <ClockCircleOutlined />
+        &nbsp; 10-30 min
+      </div>
```

Comment:
Maybe move it to a renderer function or component?
```
const : React.FC<{ minutes: string }> = ({ minutes }) => (
  <div>
    <ClockCircleOutlined />
    &nbsp; {minutes} min
  </div>
);
```

Reply:
sure, thanks.